src/SDKs/DataFactory/Management.DataFactory/changelog.md

```diff
@@ -4,3 +4,8 @@
 
 ### Feature Additions
   * Initial public release of the Azure Data Factory V2 .NET SDK.
+
+## Version 0.2.1-preview
```

Comment:
please move this section above the existing one, so the latest version is the first one from the top. thanks!